tests/functional/browser.js

```diff
@@ -0,0 +1,37 @@
+const { expect } = chai;
```

Comment:
Why do we need a special test for browsers?

Reply:
Will do the same thing we did for wallet-lib 👍